comp/core/agenttelemetry/impl/config.go

```diff
@@ -210,6 +210,7 @@ var defaultProfiles = `
         - name: logs.dropped
         - name: logs.encoded_bytes_sent
         - name: logs.sender_latency
+        - name: logs.auto_multi_line_aggregator_flush
```

Comment:
Don't you want to add list of tags, like [here](https://github.com/DataDog/datadog-agent/blob/main/comp/core/agenttelemetry/impl/config.go#L192-L194).  Details explained [here](https://github.com/DataDog/datadog-agent/blob/main/comp/core/agenttelemetry/impl/config.go#L111-L125). If you do not add "truncated" and "line_type" all timeseries will be aggregated in a single value, but that not what you want right?